### PR TITLE
brew-gem 0.7.0

### PR DESCRIPTION
**Pull request by nicksieger**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

The only issue with `brew audit --strict --online` is that sportngin/brew-gem is a forked repository, but it has been that way since before this new release.

### CODE REVIEW

File: Formula/brew-gem.rb [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-  sha256 "b3dd161d5fa6a643d97b08cad514f0c8fdba8b8f44ff80a70f6a5090cf02cdfd"
+  url "https://github.com/sportngin/brew-gem/archive/v0.7.0.tar.gz"
+  sha256 "34f5999fc2c0a99118e0419f89be35a6e42ad9abad99e1c13e34d9162121b541"
+
+  head "https://github.com/sportngin/brew-gem.git"
 
   bottle :unneeded
 
   def install
+    lib.install Dir["lib/*"]
```
Comment (mikemcquaid) [inline Formula/brew-gem.rb]: Is this neeed by `brew-gem` or is it useful standalone?

Comment (ilovezfs) [inline Formula/brew-gem.rb]: most of the internals were moved there

**Comment by ilovezfs** [comment]
@nicksieger 
```
>> `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
=> [".gitignore", ".octopolo.yml", ".rspec", ".ruby-version", ".travis.yml", "Gemfile", "LICENSE", "README.md", "Rakefile", "bin/brew-gem", "brew-gem.gemspec", "lib/brew/gem.rb", "lib/brew/gem/cli.rb", "lib/brew/gem/formula.rb.erb", "lib/brew/gem/version.rb"]
```
Do you not also want to install your LICENSE and README.md files?

**Comment by DomT4** [comment]
> Do you not also want to install your LICENSE and README.md files?

This happens automagically in a lot of cases FWIW, including here:

```
ls -1 $(brew --prefix brew-gem)
INSTALL_RECEIPT.json
LICENSE
README.md
bin
```

**Comment by ilovezfs** [comment]
interesting... where's that voodoo magic?

**Comment by DomT4** [comment]
The magic is [metafiles](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/metafiles.rb) and invoked by build [here](https://github.com/Homebrew/brew/blob/acc9a7ca8554bc2413dee2d6d0f407b3a59c628c/Library/Homebrew/build.rb#L136-L138).

**Comment by nicksieger** [comment]
Ok, I'm new to formula dev and not clear on the difference between `/usr/local/opt` (prefix) and `/usr/local/Cellar/<formula>/<version>`. I can certainly update the formula to install the license and readme to the cellar, if that's the convention.

**Comment by DomT4** [comment]
> Ok, I'm new to formula dev and not clear on the difference between /usr/local/opt (prefix) and /usr/local/Cellar/<formula>/<version>. 

The former is just an unversioned symlink to the latter, essentially.

> I can certainly update the formula to install the license and readme to the cellar, if that's the convention.

Doesn't seem necessary here. It's happening automatically.

**Comment by nicksieger** [comment]
Aha, sure enough. Thanks. Let me know if anyone sees any other issues that need resolution.

File: Formula/brew-gem.rb (hunk 1 above)
Comment (nicksieger) [inline Formula/brew-gem.rb]: See sportngin/brew-gem#23 for more information on that

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**